### DISCUSSION TITLE
PC764

### DISCUSSION THREAD

**Discussion by pc764** [remay/tmf-hudl#35] [Help]
I extract the files from hudl.20140424 using windows explorer to SD card but when I put it into hudl it says there are corrupted files. What am I doing wrong. Thanks

**Comment by remay** [comment]
Please can you be a bit more explicit about what says what files are corrupted?

Are you following the detailed steps from this page: https://github.com/remay/tmf-hudl/wiki/Flashing-from-an-SD-Card  (click on the little triangles to expand the steps for more information) Let us know which step you think you're having trouble with.

Rob.

**Comment by pc764** [comment]
I am downloading file to my download folder. I am then extracting the files using windows explorer extractor. I then click to open file and it says image file corrupted. I have never done anything like this before so it could very well be my lack of knowledge. I’m not sure what unpack file means or how to do it.

**Reply by remay** [reply]
OK.   You should have downloaded a file called `hudl.20140424.153851.TMF-Custom-v1.0.img.zip`.    "Unpacking" is (I think) what you are calling extracting.   The result of unpacking/extracting should be a (bigger) file called `hudl.20140424.153851.TMF-Custom-v1.0.img`.   Rename this file to `update.img` and then copy/move it to your micro SD card. so that your SD card has a file called `update.img` on it.

The continue following the instructions from step 3: https://github.com/remay/tmf-hudl/wiki/Flashing-from-an-SD-Card

**Comment by pc764** [comment]
Hi Robert,

 

I have followed the steps but my Hudl says: can not found firmware image or invalid image .Update rkimage complete

 

What should I do now?

 

Thanks so much for your help.

 

Best wishes,

 

Paul Cullen

**Comment by pc764** [comment]
Hi Robert,

 

If I click on the unpacked file (update.img) to open it I get a window that says The disk image file is corrupted.

 

Thanks,

 

Paul Cullen

**Comment by remay** [comment]
> If I click on the unpacked file (update.img) to open it I get a window that says The disk image file is corrupted.

That's expected.  You shouldn't be trying to open it on a Windows PC, as it's a format that only a Hudl recognises.   Windows expects a `.img` file to be a disk image, and when it isn't it reports it as corrupted. 

> I have followed the steps but my Hudl says: can not found firmware image or invalid image .Update rkimage complete

Which, most likely means that you've mis-understood the steps somewhere along the line.   The most likely cause of that message is that the file `update.img` is not found on the SD card, or that the file is not what's expected.

Try this:
1. Download this version of `update.img` from WeTransfer: https://we.tl/t-l3Hgk9fMYX. (That's just the content of the download from Github that I've unpacked and renamed for you).
2. Copy it onto your SD Card
3. Try again from step 3 https://github.com/remay/tmf-hudl/wiki/Flashing-from-an-SD-Card

Let us know how you get on.

Rob.

**Comment by pc764** [comment]
Hi Robert,

 

My PC is saying that your file is dangerous and do not download. I entered keep but the file is no longer update.img.

It reads unconfirmed 196129.crdownload.

I’ve sent to sd card but again does not like this.

 

Best wishes,

 

Paul Cullen

**Comment by pc764** [comment]
I have changed the name of the file to update.img but again hudl doesn’t accept it.
Best wishes,
Paul Cullen

**Comment by pc764** [comment]
Hi Robert,

 

Downloaded your file again this morning and windows accepted it okay. Put it into  Hudl and it accepted the update and is now working again.

 

Thank you so much. Absolutely brilliant.

 

Best wishes,

 

Paul Cullen

**Reply by remay** [reply]
> My PC is saying that your file is dangerous and do not download.

I can assure you that it's fine.

> I entered keep but the file is no longer update.img.
> It reads unconfirmed 196129.crdownload.

With a filename like `unconfirmed 196129.crdownload` it looks like you are using Chrome browser to download.  That's the temporary filename the browser uses until download is complete at which point it should get renamed `update.img`.   This suggests you didn't wait for the download to complete, or the download failed.   I suggest deleting anything you downloaded and trying again.

> Downloaded your file again this morning and windows accepted it okay. Put it into  Hudl and it accepted the update and is now working again.

Ah.  Brilliant.   Should have read that first before starting to respond.  Glad to have got you working.

Rob.

**Comment by pc764** [comment]
Thank you so much Rob.

Best wishes,

Paul Cullen
